activerecord/CHANGELOG.md

```diff
@@ -1 +1,715 @@
-Please check [5-1-stable](https://github.com/rails/rails/blob/5-1-stable/activerecord/CHANGELOG.md) for previous changes.
+*   Fix circular `autosave: true`
```

Comment:
Looks like you rebase pulled more lines that we need. Could you keep only the "Please check [5-1-stable](https://github.com/rails/rails/blob/5-1-stable/activerecord/CHANGELOG.md) for previous changes." and you new entry?